### DISCUSSION TITLE
How many hosts do you have?

### DISCUSSION THREAD

**Discussion by antonmedv** [deployphp/deployer#3917] [General]
How many hosts do you have? Do you rely on parallel deployment of hosts? I'm thinking to remove this functionality. What do you think?

**Comment by M-arcus** [comment]
> How many hosts do you have?

2+ per project, Staging and Production (* load balancer). In case of big features (like relaunches), sometimes 3 (* load balancer on prod).

> Do you rely on parallel deployment of hosts?

If the project requires it, yes.

> I'm thinking to remove this functionality. What do you think?

Is the functionality negatively affecting deployer, and it's development? (honest question, so far I have been focussing mostly on improvements of recipes, which were never negatively affected by the parallel deployment)

**Reply by antonmedv** [reply]
It complicated things a little bit.

**Comment by Aquive** [comment]
At least 2. Why do you want to remove it?

**Reply by antonmedv** [reply]
To make development easier. But if people are using this feature - will keep it.

**Comment by mrhackcz** [comment]
Please no. We have 3 production app servers (cluster) + 1 staging. Using Deployer with parallel deploy, sped up the release from 2 minutes to 20 secs. Love that.

**Reply by antonmedv** [reply]
Nice! Got it. Thanks for sharing.

**Comment by antonmedv** [comment]
Thanks for your feedback. I will keep the feature.
